fission/src/ui/panels/configuring/assembly-config/interfaces/ConfigureGamepiecePickupInterface.tsx

```diff
@@ -107,7 +112,7 @@ const ConfigureGamepiecePickupInterface: React.FC<ConfigPickupProps> = ({ select
             return
         }
 
-        transformGizmo.mesh.scale.set(zoneSize, zoneSize, zoneSize)
+        transformGizmo.obj.scale.set(zoneSize, zoneSize, zoneSize)
```

Comment:
I don't like that we're manipulating internal gizmo objects from outside of it. Maybe there should be a method to set the scale of the gizmo from out here? That way if the implementation of the gizmo changes, we would only need to make sure all the methods inside the class still work as expected, rather than needing to track down all the places where we made assumptions like this about the internal implementation of the class.